dash_cytoscape/CyLeaflet.py

```diff
@@ -212,10 +205,68 @@ def get_cytoscape_max_zoom(self, leaflet_max_zoom):
             {"id": MATCH, "component": "cyleaflet", "sub": "leaf"}, "invalidateSize"
         ),
         Output({"id": MATCH, "component": "cyleaflet", "sub": "leaf"}, "viewport"),
+        Output({"id": MATCH, "component": "cyleaflet", "sub": "cy-extent"}, "data"),
         Input({"id": MATCH, "component": "cyleaflet", "sub": "cy"}, "extent"),
+        Input({"id": MATCH, "component": "cyleaflet", "sub": "cy"}, "maxZoom"),
+        State({"id": MATCH, "component": "cyleaflet", "sub": "cy-extent"}, "data"),
     )
     clientside_callback(
         ClientsideFunction(namespace="cyleaflet", function_name="transformElements"),
-        Output({"id": MATCH, "component": "cyleaflet", "sub": "cy"}, "elements"),
+        Output(
+            {"id": MATCH, "component": "cyleaflet", "sub": "cy"},
+            "elements",
+            allow_duplicate=True,
+        ),
         Input({"id": MATCH, "component": "cyleaflet", "sub": "elements"}, "data"),
+        prevent_initial_call="initial_duplicate",
     )
+
+
+@callback(
+    Output({"id": MATCH, "component": "cyleaflet", "sub": "elements"}, "data"),
+    Input({"id": MATCH, "component": "cyleaflet", "sub": "cy"}, "elements"),
+    prevent_initial_call=True,
+)
+def update_elements_store(elements):
```

Comment:
Is there a reason this can't be a clientside callback instead? It would be nice to avoid the round-trip to the server every time the user drags an element (although maybe not a huge deal in practice).

Reply:
And would also mean we don't have to maintain two copies of the conversion function (one in Python and one in Javascript), we could just use the Javascript one.

Reply:
Good point! Moved to clientside callbacks